gdal/doc/source/api/index.rst

```diff
@@ -82,6 +82,11 @@ API
    `Java API <../java/index.html>`_
    --------------------------------
 
+   .. toctree::
```

Comment:
shouldn't that be moved right under the Python API paragraph ? In which case, we might want to remove the link from the Python API title itself, to have 2 entries, one that could be "API reference" and use the current <../python/index.html> and the other the python_gotchas . But not completely sure we can do that that way...

Reply:
Yes, I wasn't totally sure where the best spot for it would be but that makes sense. I'll take a look.